spec/controllers/api/v1/operators_controller_spec.rb

```diff
@@ -82,18 +82,18 @@
 
   describe 'GET show' do
     context 'as JSON' do
-      it 'returns operators by OnestopId' do
+      it 'returns operators by Onestop ID' do
         get :show, id: @vta.onestop_id
-        expect_json_types({
+        expect_json_types('operator',
           onestop_id: :string,
           geometry: :object,
           name: :string,
           created_at: :date,
           updated_at: :date
-        })
-        expect_json({ onestop_id: -> (onestop_id) {
+        )
+        expect_json('operator', onestop_id: -> (onestop_id) {
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Use the `lambda` method for multi-line lambdas.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.